analyses/NC_cd_2010/01_prep_NC_cd_2010.R

```diff
@@ -0,0 +1,83 @@
+###############################################################################
+# Download and prepare data for `NC_cd_2010` analysis
+# © ALARM Project, April 2022
+###############################################################################
+
+suppressMessages({
+    library(dplyr)
+    library(readr)
+    library(sf)
+    library(redist)
+    library(geomander)
+    library(cli)
+    library(here)
+    devtools::load_all() # load utilities
+})
+
+# Download necessary files for analysis -----
+cli_process_start("Downloading files for {.pkg NC_cd_2010}")
+
+path_data <- download_redistricting_file("NC", "data-raw/NC", year = 2010)
+
+# download the enacted plan.
+# 2010 plan from https://redistricting.lls.edu/state/north-carolina/
+path_enacted <- "data-raw/NC/NC_enacted/Rucho_Lewis_Congress_3.shp"
```

Comment:
We should probably stick to the same downloading/unzipping template here.
```
url <- "___insert_zip_download_url___"
path_enacted <- "data-raw/NC/NC_enacted.zip"
download(url, here(path_enacted))
unzip(here(path_enacted), exdir = here(dirname(path_enacted), "NC_enacted"))
file.remove(path_enacted)
```

Reply:
Thinking a bit about this, we may be able to standardize this to avoid any shape downloads, as the CB has collected these. I'll circle back with the `make_from_baf` points, as it's likely a connected fix.